RBDM/Table.cpp

```diff
@@ -0,0 +1,108 @@
+#include"Table.h"
+
+Table::Table()
+{
+	id = ++TID;
+	name = "table " + std::to_string(id);
+}
+void Table::ArrRecord()
+{
+	Record *rec = new Record();
+	char type;
+	int id;
+	while (true)
+	{
+		std::cin >> type;
+		if (type == 'i')
+		{
+			long value;
+			std::cin >> value;
+			if (!std::cin.fail())
+			{
+				Integer *arg = new Integer(value);
+				id = table.size() + 1;
```

Comment:
I think this thing will give us some set of records with equal id.
Please, give each record unique id instead.